src/components/ui/Breadcrumb/Breadcrumb.tsx

```diff
@@ -13,17 +15,17 @@ export interface BreadcrumbProps extends UIBreadcrumbProps {
   breadcrumbList: ItemElement[]
 }
 
-function Breadcrumb({ breadcrumbList, divider }: BreadcrumbProps) {
+function Breadcrumb({ breadcrumbList }: BreadcrumbProps) {
   const buildUrl = (url: string) => {
     const parsedUrl = url.split('/')
 
     return `/${parsedUrl[parsedUrl.length - 1]}`
   }
 
   return (
-    <UIBreadcrumb divider={divider}>
+    <UIBreadcrumb divider="">
```

Comment:
I noticed that when we pass a divider here something strange happens

<img width="197" alt="Screen Shot 2021-12-30 at 20 28 57" src="https://user-images.githubusercontent.com/11325562/147794763-5bb37271-5c1c-4ce0-88e8-5e148c6d0eef.png">

Reply:
The default divider of this component is "/", this need be changed in faststore, but to our use divider="" resolved this problem